doc/decoding/searchlight.rst

```diff
@@ -91,46 +89,52 @@ Cross validation
-
-.. literalinclude:: ../../examples/02_decoding/plot_haxby_searchlight.py
-    :start-after: # set once and the others as learning sets
-    :end-before: import nilearn.decoding
-
-Running Searchlight
-===================
-
-Running :class:`SearchLight` is straightforward now that everything is set.
-The only
-parameter left is the radius of the ball that will run through the data.
-Kriegskorte et al. use a 5.6mm radius because it yielded the best detection
-performance in their simulation.
-
-.. literalinclude:: ../../examples/02_decoding/plot_haxby_searchlight.py
-    :start-after: cv = KFold(n_splits=4)
-    :end-before: # F-scores computation
+this score as accurate as possible (and avoid overfitting), cross-validation
+is used.
+
+Cross-validation can be defined using the "cv" argument. As it
+is computationally costly, *K*-Fold cross validation with *K* = 3 is set as the 
+default. A `scikit-learn cross-validation generator 
+<https://scikit-learn.org/stable/modules/classes.html#splitter-classes>`_ can also 
+be passed to set a specific type of cross-validation. 
+
+Leave-one-run-out cross-validation (LOROCV) is a common approach for searchlights. 
+This approach is a specific use-case of grouped cross-validation, where the 
+cross-validation folds are determined by the acquisition runs. The held-out fold
+in a given iteration of cross-validation consist of data from a separate run, 
+which keeps training and validation sets properly independent. For this reason, 
+LOROCV is often recommended. This can be performed by using `LeaveOneGroupOut 
+<https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.LeaveOneGroupOut.html>`_,
+and then setting the group/run labels when fitting the estimator. 
+
+Sphere radius
+-------------
+
+An important parameter is the radius of the sphere that will run through 
+the data. The sphere size determines the number of voxels/features to use 
+for classification (i.e. more voxels are included with larger spheres). 
+
+.. note::
+    :class:`SearchLight` defines sphere radius in milimeters; the number 
+    of voxels included in the sphere will depend on the voxel size. 
+
+For reference, Kriegskorte et al. use a 4mm radius because it yielded 
+the best detection performance in their simulation of 2mm isovoxel data.
 	
 Visualization
 =============
 
 Searchlight
 -----------
 
-As the activation map is cropped, we use the mean image of all scans as a
-background. We can see here that voxels in the visual cortex contains
-information to distinguish pictures showed to the volunteers, which was the
+The results of the searchlight can be found in the `scores_` attribute of the 
+:class:`SearchLight` object after fitting it to the data. Below is a 
+visualization of the results from :ref:`Searchlight analysis of face 
+vs house recognition <sphx_glr_auto_examples_02_decoding_plot_haxby_searchlight_py>`.
+We can see here that voxels in the visual cortex contains information 
```

Comment:
Isn't the process mask restricted to visual cortex, though ? Should we clarify this ?